Alignment/CommonAlignmentProducer/python/ALCARECOMuAlGlobalCosmicsInCollisions_cff.py

```diff
@@ -7,29 +6,45 @@
     andOr = True, ## choose logical OR between Triggerbits
     eventSetupPathsKey = 'MuAlGlobalCosmics',
     throw = False # tolerate triggers not available
-    )
+)
 
 # DCS partitions
 # "EBp","EBm","EEp","EEm","HBHEa","HBHEb","HBHEc","HF","HO","RPC"
 # "DT0","DTp","DTm","CSCp","CSCm","CASTOR","TIBTID","TOB","TECp","TECm"
 # "BPIX","FPIX","ESp","ESm"
 import DPGAnalysis.Skims.skim_detstatus_cfi
 ALCARECOMuAlGlobalCosmicsInCollisionsDCSFilter = DPGAnalysis.Skims.skim_detstatus_cfi.dcsstatus.clone(
-    DetectorType = cms.vstring('DT0','DTp','DTm'),
+    DetectorType = cms.vstring('DT0','DTp','DTm',"CSCp","CSCm"),
     ApplyFilter  = cms.bool(True),
     AndOr        = cms.bool(False),
     DebugOn      = cms.untracked.bool(False)
 )
 
+#________________________________Muon selection____________________________________
+# AlCaReco selected muons for track based muon alignment
 import Alignment.CommonAlignmentProducer.AlignmentMuonSelector_cfi
-
 ALCARECOMuAlGlobalCosmicsInCollisions = Alignment.CommonAlignmentProducer.AlignmentMuonSelector_cfi.AlignmentMuonSelector.clone(
-    src = cms.InputTag("muonsFromCosmics"),
-    filter = True, # not strictly necessary, but provided for symmetry with MuAlStandAloneCosmics
-    nHitMinGB = 1,
-    ptMin = 10.0,
-    etaMin = -100.0,
-    etaMax =  100.0
-    )
+    src = cms.InputTag("muons"),
+    filter = cms.bool(True), # not strictly necessary, but provided for symmetry with MuAlStandAloneCosmics
+    ptMin = cms.double(10.0),
+    etaMin = cms.double(-100.0),
+    etaMax = cms.double(100.0),
+)
+
+#________________________________Track selection____________________________________
+# AlCaReco selected general tracks for track based muon alignment
+import Alignment.CommonAlignmentProducer.AlignmentTrackSelector_cfi
+ALCARECOMuAlGlobalCosmicsInCollisionsGeneralTracks = Alignment.CommonAlignmentProducer.AlignmentTrackSelector_cfi.AlignmentTrackSelector.clone(
+    src = cms.InputTag("generalTracks"),
```

Comment:
@mmusich, that was exactly the purpose of this producer: "get cosmic tracks collected during collisions". This is just for potential tests, which we didn't used in the past. What we used was just a normal cosmics. So, if there is any doubt about this producer we could remove it to push the PR through.